### PR TITLE
Update webhooks documentation to include document upload

### PR DESCRIPTION
**Pull request by sampras0**
CIA-6391

- updated existing image upload to file upload for images and documents

### CODE REVIEW

File: docs/webhooks/create-haptik-user.md [hunk 1]
```diff
@@ -41,7 +41,6 @@ Content-Type: application/json
 ```json
 {
   "auth_id": "<AUTH_ID>",
-  "auth_code": "<AUTH_CODE>",
```
Comment (tejas-haptik) [inline docs/webhooks/create-haptik-user.md:44]: @sampras0 we removed this on purpose?

File: docs/webhooks/send-file-to-haptik.md [hunk 2]
```diff
@@ -1,64 +1,48 @@
...
-
-<AUTH_ID>
---MultipartBoundry
-Content-Disposition: form-data; name="business_id"
-Content-Length: 3
-
-<BUSINESS_ID>
---MultipartBoundry
-Content-Disposition: form-data; name="message_type"
-Content-Length: 1
```
Comment (tejas-haptik) [inline docs/webhooks/send-file-to-haptik.md:40]: Here we did mention the `message_type` in the request which is missing from the current documentation

File: docs/webhooks/send-file-to-haptik.md [hunk 3]
```diff
@@ -77,8 +61,8 @@ A successful request will return a `200` status code with a JSON response object
 - message_id - message id generated by haptik system
 - message_body - message body that was logged in the haptik system
 - created_at - ISO timestamp denoting when the message was created in the haptik system
-- message_type - The message type should be `1` for image
-- file_url - The url for the uploaded image
+- message_type - The message type should be `1` for image and `52` for document
```
Comment (tejas-haptik) [inline docs/webhooks/send-file-to-haptik.md:64]: This should also be mentioned in the section which specifies how to make requests

File: docs/webhooks/send-file-to-haptik.md [hunk 4]
```diff
@@ -1,64 +1,48 @@
...
+  "file": "<FILE>"
+}
 ```
 
 - auth_id - This is an alphanumeric User identifier from your system
 - business_id - This is a numeric identifier for channel/queue on which the message is to be registered.
-- message_type - The message type should be `1` for image
-- file - contents of the image (Supported extensions: jpeg, png). Max file size allowed is 10MB
-- message_body - message body that was logged in the haptik system
+- message_type - The message type should be `1` for image and `52` for document
```
Comment (tejas-haptik) [inline docs/webhooks/send-file-to-haptik.md:39]: Can we highlight the message types a little bit? Bullet list maybe? @sampras0

File: docs/webhooks/using-platform-deployments.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-Title: Using Platform Deployments
+title: Using Platform Deployments
```
Comment (tejas-haptik) [inline docs/webhooks/using-platform-deployments.md:2]: Nice. Thanks 👍

File: docs/webhooks/create-haptik-user.md (hunk 1 above)
Comment (sampras0) [reply docs/webhooks/create-haptik-user.md:44]: there is no `auth_code` used in `message_delivery`.

Comment (tejas-haptik) [reply docs/webhooks/create-haptik-user.md:44]: So it's only used for user creation?

File: docs/webhooks/send-file-to-haptik.md (hunk 2 above)
Comment (sampras0) [reply docs/webhooks/send-file-to-haptik.md:40]: The current document also has the request body parameter `message_type` on line 31.
but the new request body is json formatted to resemble other pages.

File: docs/webhooks/create-haptik-user.md (hunk 1 above)
Comment (sampras0) [reply docs/webhooks/create-haptik-user.md:44]: `auth_id` is used for user creation.
`auth_code` is used nowhere

Comment (tejas-haptik) [reply docs/webhooks/create-haptik-user.md:44]: Got it

Comment (tejas-haptik) [inline docs/webhooks/create-haptik-user.md:44]: Good catch 👍

**Pull request merged**